src/modules/interface/controller_indi.h

```diff
@@ -43,13 +44,13 @@
 #define STABILIZATION_INDI_FILT_CUTOFF_R STABILIZATION_INDI_FILT_CUTOFF
 
 // these parameters are used in the filtering of the angular acceleration
-#define STABILIZATION_INDI_G1_P 0.0034724f
-#define STABILIZATION_INDI_G1_Q 0.0052575f
-#define STABILIZATION_INDI_G1_R -0.0015942f
-#define STABILIZATION_INDI_G2_R -0.11281f
-#define STABILIZATION_INDI_REF_ERR_P 3.57f
-#define STABILIZATION_INDI_REF_ERR_Q 3.57f
-#define STABILIZATION_INDI_REF_ERR_R 3.57f
```

Comment:
This is quite a difference. How did you tune this? I guess this are the values for the INDI attitude controller right?

Reply:
The attitude INDI was quite uncontrollable (very slow and not very stable), that's why we have decided to re-estimate some of the parameters. 
The gains (`STABILIZATION_INDI_REF_ERR_P/Q/R`) were tuned manually. The control effectiveness parameters (G1, G2) of the INDI were tuned according to the paper "Adaptive Incremental Nonlinear Dynamic Inversion for Attitude Control of Micro Air Vehicles" (Section A. Parameter Estimation). I have uploaded two Matlab scripts (`param_estimation.m`, `import_logdata.m`) and the flight data (`data_1901.csv`) I used to estimate these parameters. They are located in the `/tools/param_est` folder.